doc/conf.py

```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'FOSHttpCache'
-copyright = '2014, FriendsOfSymfony'
+copyright = u'2014–2016, FriendsOfSymfony'
```

Comment:
> why the "u"? 

This makes it a Unicode string (in Python/Sphinx). We need that for the dash, as it’s an [EN dash](http://www.thepunctuationguide.com/en-dash.html), not a hyphen, between a range of numbers. Sorry about that, I’m a typography nerd. :wink:

>  could we just remove the year as its not covered+updated by styleci?

Does StyleCI update the years somewhere?